docs/introduction.rst

```diff
@@ -35,11 +35,27 @@ First steps
 Getting involved
 ----------------
 
+``stdpopsim`` **is a community effort, and we welcome YOU to join us!**
+
 Are there other features, models or organisms that you'd like to see in ``stdpopsim``?
 This software is maintained by the PopSim Consortium,
-a global community of scientists and developers who are working together to improve
+an open, global community of scientists and developers who are working together to improve
 standards for benchmarking and testing in population genetics.
-Read the :ref:`Development <sec_development>` page to find out how you can join us.
+You can get into contact with the ``stdpopsim`` community by subscribing to our `email list
+serve <https://lists.uoregon.edu/mailman/listinfo/popgen_benchmark>`_
```

Comment:
"email list serve". I guess this should be "server", but maybe just "email list" is sufficient?